Dockerfile

```diff
@@ -9,11 +9,11 @@ RUN apt-get update && apt-get install -y \
 RUN git clone https://github.com/jnovy/pxz.git /root/pxz
 RUN cd /root/pxz && make
 
-FROM ubuntu:19.10
+FROM ubuntu:20.04
```

Comment:
Thought:
Why not just go with `latest` here? There's not really much that could break I think.

Reply:
Sounds risky, `2020.04` is an LTS, so I guess we should be fine for a while.

```
```

Reply:
We should stick to stable releases I think, and 20.04 is an LTS. That's one of the reasons for using a container in the first place. `mkfs.ext4` has actually broken several times in the past.

Reply:
haha, I knew this was gonna get downvoted to hell. So risk averse!

Reply:
Working too long in release management! ;-)

Reply:
So you are basically just trolling @oleorhagen ?

Reply:
Just a reminder that I am yet young